.github/workflows/build.yml

```diff
@@ -12,7 +12,12 @@ jobs:
       packages: write
       contents: read
     env:
-      IMAGE_NAME: bug-buster-machine
+      PNPM_VERSION: 9
+      NODE_VERSION: 18
+      CARTESI_CLI_VERSION: 0.16.0
+      MACHINE_IMAGE_NAME: bug-buster-machine
+      NODE_IMAGE_NAME: bug-buster-machine
```

Comment:
```suggestion
      NODE_IMAGE_NAME: bug-buster-node
```

Reply:
Maybe `ROLLUPS_NODE_IMAGE_NAME` or `CARTESI_NODE_...`, reading it now, the `NODE_` prefix made me refer to the `NODE_VERSION` variable.

Reply:
> the `NODE_` prefix made me refer to the `NODE_VERSION` variable.

True!